### PR TITLE
Migrating from travis ci to github actions

### PR DESCRIPTION
**Pull request by sujith-mn**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up node
+        uses: actions/setup-node@v1
+      - name: Install dependencies with yarn
+        run: npm install
+#      - name: Test
```
Comment (maybeec) [inline .github/workflows/release.yml]: Wrong indentations

**Review by maybeec — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (sujith-mn) [reply .github/workflows/release.yml]: it was a commented line.. but removed it now.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up node
+        uses: actions/setup-node@v1
+      - name: Install dependencies with yarn
```
Comment (maybeec) [inline .github/workflows/release.yml]: please properly name the step - here dependencies are not installed by yarn, but by npm!

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+        run: npm install
```
Comment (maybeec) [inline .github/workflows/release.yml:12]: please add test execution

**Review by maybeec — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 3 above)
Comment (sujith-mn) [reply .github/workflows/release.yml:12]: @maybeec,  done with all the review comments. Could you check.

**Review by maybeec — APPROVED** [review]

**Pull request merged**